# Flight Control
uint8 control_flag
float32 x
float32 y
float32 z
float32 yaw



